packages/gateway/src/utils/redis.ts

```diff
@@ -19,12 +20,13 @@ export async function stop() {
   redisClient.quit()
 }
 
-export function start(host = REDIS_HOST, port?: number) {
+export function start(host = REDIS_HOST, password = REDIS_PASSWORD, port?: number, ) {
   return new Promise<redis.RedisClient>((resolve) => {
     logger.info(`REDIS_HOST, ${JSON.stringify(host)}`)
     redisClient = redis.createClient({
       host: host,
       port,
+      password: password,
```

Comment:
```suggestion
      host,
      port,
      password,
```

shorthand works here